packages/server/lib/controllers/sync.controller.ts

```diff
@@ -425,12 +425,11 @@ class SyncController {
                 return;
             } else {
                 span.setTag('nango.error', actionResponse.error);
-                errorManager.errResFromNangoErr(res, actionResponse.error);
                 await logCtx.error('Failed to trigger action', { error: actionResponse.error });
                 await logCtx.failed();
-                span.finish();
 
-                next(actionResponse.error);
```

Comment:
completely missed that errorManager.errResFromNangoErr was already sending the json, honestly confusing, hopefully with the refactor of the API it's going to be smoother